### PR TITLE
Fix depreciation warnings

### PR DESCRIPTION
**Pull request by roschaefer**
I couldn't see any difference in using `transition`, `pageTransition` or
neither of those, so I decided to disable the configuration completely.

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: webapp/nuxt.config.js [hunk 1]
```diff
@@ -21,10 +21,10 @@ module.exports = {
...
   modern: !dev ? 'server' : false,
 
-  transition: {
-    name: 'slide-up',
-    mode: 'out-in',
-  },
+  // transition: {
+  //  name: 'slide-up',
+  //  mode: 'out-in',
+  // },
```
Comment (mattwr18) [inline webapp/nuxt.config.js]: is there a reason we aren't either deleting this, or changing it to `pageTransition`?

File: webapp/nuxt.config.js [hunk 2]
```diff
@@ -162,7 +162,9 @@ module.exports = {
   // Give apollo module options
   apollo: {
     tokenName: 'human-connection-token', // optional, default: apollo-token
-    tokenExpires: 3, // optional, default: 7 (days)
+    cookieAttributes: {
+      expires: 3, // optional, default: 7 (days)
+    },
```
Comment (mattwr18) [inline webapp/nuxt.config.js]: is this related to a different deprecation warning that I'm not aware of? or how is this related?

Comment (roschaefer) [reply webapp/nuxt.config.js]: Err, yes, if you `yarn run dev` you see a deprecation warning of `nuxt-apollo` about the `tokenExpires`. They just added more options and want to put it into an object

File: webapp/nuxt.config.js (hunk 1 above)
Comment (roschaefer) [reply webapp/nuxt.config.js]: My motivation to disable, not remove it: I could not see any difference in removing or adding it. However it is a documented feature. Maybe w'ere doing something wrong? Maybe I'm blind?

I can also remove it, if that helps you @mattwr18

File: webapp/nuxt.config.js (hunk 2 above)
Comment (mattwr18) [reply webapp/nuxt.config.js]: thanks for the explanation :)

File: webapp/nuxt.config.js (hunk 1 above)
Comment (mattwr18) [reply webapp/nuxt.config.js]: well, I don't think it would help me so much personally, hehe, the reason I ask is because it can be easy to collect comments in the code base and if we don't want/need some code, let's delete it, I think.

I pushed up some changes, if you can review. 

I had a look at the docs, and it seems like this config would be used anytime we have a `slide-up` transition, which we have a handful of, so I think it might be best to leave it, not remove it.
https://nuxtjs.org/api/pages-transition/
https://vuejs.org/v2/guide/transitions.html#Transition-Modes
